### PR TITLE
Make DoubleMatrix type available within cookbooks.

### PR DESCRIPTION
**Pull request by arianepaola**
This adds the DoubleMatrix type and makes it available within cookbooks.

### CODE REVIEW

File: examples/meta/generator/targets/cpp.json [hunk 1]
```diff
@@ -45,7 +45,8 @@
         "ShortRealMatrix": "SGMatrix<float32_t>",
         "RealMatrix": "SGMatrix<float64_t>",
         "LongRealMatrix": "SGMatrix<floatmax_t>",
-        "ComplexMatrix": "SGMatrix<complex128_t>"
+        "ComplexMatrix": "SGMatrix<complex128_t>",
+        "DoubleMatrix": "SGMatrix<float64_t>"
```
Comment (vigsterkr) [inline examples/meta/generator/targets/cpp.json]: DoubleMatrix is RealMatrix, see above...

**Comment by vigsterkr** [comment]
use `RealMatrix`

**Comment by arianepaola** [comment]
ok

**Comment by arianepaola** [comment]
@vigsterkr @karlnapf @lgoetz

Ok, also as noted by @vigsterkr

**Pull request closed without merging**